task_sdk/tests/api/test_client.py

```diff
@@ -133,3 +134,21 @@ def handle_request(request: httpx.Request) -> httpx.Response:
                 "reason": "not_found",
             }
         }
+
+    def test_variable_set_success(self):
+        # Simulate a successful response from the server when putting a variable
+        def handle_request(request: httpx.Request) -> httpx.Response:
+            if request.url.path == "/variables/test_key":
+                return httpx.Response(
+                    status_code=200,
+                    json={"ok": True},
```

Comment:
This doesn't matter but server response is `{"message": "Variable successfully set"}` :D

Reply:
Yeah, better to stick to actuals.